samples/GuestConfiguration/audit-installed-application-linux/README.md

```diff
@@ -0,0 +1,42 @@
+# Audit for installed application - Linux
+
+This policy audits the results of a check for applications installed in Linux virtual machines.
+
+This policy requires another definition to also be deployed that loads requirements for performing audits inside virtual machines.  Please see the policy sample
+[Deploy extension to audit for installed applications - Linux](..\deployextension-installed-application-linx\README.md).
+
+## Azure Policy Docs
+
+See more information and a complete walk-through of using this sample on
+[docs.microsoft.com](https://docs.microsoft.com/azure/governance/policy/samples/audit-installed-application-linux).
```

Comment:
will this be added to the samples page once it is published here?

Reply:
Once they are merged, I'll be creating a Samples page in docs for them. Still working on the template for how a GC policy set of defs will look.